### ISSUE TITLE
iinstall - launch the app after installing

### ISSUE THREAD

**Issue by IGPenguin** [IGPenguin/mobile-toolkit#91]
Following code is "working" but the app will crash almost always in few seconds after launch, I can:
- launch it multiple times to boost the odds for app to remain running (ugly) [libimobiledevice Issue](https://github.com/libimobiledevice/libimobiledevice/issues/535)
- try to find another tool for launching iOS apps using bundle id (didnt find any yet)

```
echo "🔍 Detecting bundle id..."
unzip "$PWD/$1" &> /dev/null
APP_FILENAME=$(ls $PWD/Payload)
BUNDLE_ID=$(grep "$PWD/Payload/$APP_FILENAME/embedded.mobileprovision" -a -e "[.]com[.]" | sed 's/.[^.]*\.//' | sed 's/\<.*$//')
rm -rf "$PWD/Payload"
echo "⌛️ Installing $BUNDLE_ID"
ideviceinstaller -i "$1" &> /dev/null #make silent
echo "🚀 Launching..."
idevicedebug run $BUNDLE_ID
```

**Comment by neildufva** [comment]
I would really be interested in finding a way as well! I’ve been searching for months.

**Comment by radimvaculik** [comment]
@IntergalacticPenguin There is a new option `--detach` in `idevicedebug`. Check it out https://github.com/libimobiledevice/libimobiledevice/commit/fad6a83086d39d4f6ee6cab8276238a71cd165ff

**Comment by IntergalacticPenguin** [comment]
> @IntergalacticPenguin There is a new option `--detach` in `idevicedebug`. Check it out [libimobiledevice/libimobiledevice@fad6a83](https://github.com/libimobiledevice/libimobiledevice/commit/fad6a83086d39d4f6ee6cab8276238a71cd165ff)

Thank you very much! Ill look into it later today and perhaps release new version. 🎉

**Comment by IntergalacticPenguin** [comment]
A lot of changes implemented and available on libimobiledevice GitHub is not released and available on Homebrew yet, therefore I cannot implement, test and release the new functionality properly.

I have contacted the developers using Twitter, I hope they will react soon.
https://twitter.com/eidamsvoboda/status/1347229772447887360?s=20

**Comment by IntergalacticPenguin** [comment]
A lot of changes implemented and available on libimobiledevice GitHub is not released and available on Homebrew yet, therefore I cannot implement, test and release the new functionality properly. Nevertheless I drafted a [pull request](https://github.com/IntergalacticPenguin/mobile-toolkit/pull/210) resolving this issue.

I contacted the developers using Twitter, I hope they will react soon.
https://twitter.com/eidamsvoboda/status/1347229772447887360?s=20
